cloudsupport/v1/ekssupport.go

```diff
@@ -107,3 +136,24 @@ func (eksSupport *EKSSupport) GetContextName(cluster string) string {
 	}
 	return ""
 }
+
+// Getcfgmap returns the authdata containing mappings of iam-roles/groups or iam-users/groups
+func (EKSSupport *EKSSupport) GetAwsCfgMap(kapi *k8sinterface.KubernetesApi, namespace string) error {
+
+	resp, err := kapi.KubernetesClient.CoreV1().ConfigMaps(namespace).Get(context.TODO(), awsauthconfigmap, metav1.GetOptions{})
+
+	if err != nil {
+		return fmt.Errorf("aws-auth config map not found")
+	}
+
+	if err := json.Unmarshal([]byte(resp.Data["mapRoles"]), &ListmapRoles); err != nil {
```

Comment:
You should define a struct with `mapRoles` and `mapUsers` and unmarshal the response object to that struct.

Reply:
Ok will do this